aries-site/src/examples/templates/page-layouts/components/PageContainer.js

```diff
@@ -10,7 +10,10 @@ export const pageContainer = {
   wide: {
     align: 'center',
     width: {
-      max: 'xxlarge', // 1536 --> needs to adjust to 1536 + 72
+      // 1536 --> Figma shows 1608 (1536 + 72). I think this should
+      // stay at 1536 where the margins are 48 (1440 + 48 + 48) or
+      // be modified to be 1536 + 48 + 48
```

Comment:
I am good leaving as is for now, but would be good to loop Aditi in on this from the Figma side just to make sure we are aligned (both in shared mindsets but also any visuals).

Reply:
Check in is already in progress. Validating in Figma.